portfolio/src/main/java/com/google/sps/filters/ObjectifyWebFilter.java

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.filters;
+
+import javax.servlet.annotation.WebFilter;
+import com.googlecode.objectify.ObjectifyFilter;
+
+/** Filter needed by objectify. */
```

Comment:
I would elaborate a bit more what this is needed for. I would now have to dig into the docs to understand what this does.